test/fixtures/cdb_crankshaft/cdb_areasofinterestlocal.sql

```diff
@@ -0,0 +1,23 @@
+CREATE SCHEMA IF NOT EXISTS cdb_crankshaft;
+
+CREATE OR REPLACE FUNCTION
+  cdb_crankshaft.CDB_AreasOfInterestLocal(
+      subquery TEXT,
+      column_name TEXT,
+      w_type TEXT,
+      num_ngbrs INT,
+      permutations INT,
+      geom_col TEXT,
+      id_col TEXT)
+RETURNS TABLE (
+    moran NUMERIC,
+    quads TEXT,
+    significance NUMERIC,
+    rowid INT,
+    vals NUMERIC)
+AS $$
+BEGIN
+    EXECUTE subquery;
+    SELECT 1, 'QUADS', 0.01, 1, 1;
+END
+$$ LANGUAGE plpgsql VOLATILE PARALLEL UNSAFE;
```

Comment:
Tests are failing because of the `PARALLEL UNSAFE;`. We strip it in the crankshaft builild script if the DB doesn't support it, but not here. Simply removing it here would be ok.